README.md

```diff
@@ -1,21 +1,9 @@
-| Packaging | ![José Armando García Sancio](https://avatars0.githubusercontent.com/u/794394?v=3&s=35) José Armando García Sancio, Mesosphere  <br> [email](mailto:jose@mesosphere.io), [Twitter](https://twitter.com/jagsancio), [GitHub](https://github.com/jsancio) <br> ![Ravi Yadav](https://avatars2.githubusercontent.com/u/1666717?v=3&s=35) Ravi Yadav, Mesosphere <br> [email](mailto:ryadav@mesosphere.io), [Twitter](https://twitter.com/RaaveYadav), [GitHub](https://github.com/ryadav88) | #packaging-wg | [Google Group](https://groups.google.com/a/dcos.io/forum/#!forum/packaging-wg) |TBD|
-| SDK | ![Chris Lambert](https://avatars3.githubusercontent.com/u/19239758?v=3&s=35) Chris Lambert, Mesosphere <br> [email](mailto:clambert@mesosphere.io), [Twitter](https://twitter.com/clambert), [GitHub](https://github.com/triclambert) | #sdk-wg | [Google Group](https://groups.google.com/a/dcos.io/forum/#!forum/sdk-wg) | TBD |
-| UX | ![Lee Munroe](https://avatars3.githubusercontent.com/u/15963?v=3&s=35) Lee Munroe, Mesosphere <br> [email](mailto:lee@mesosphere.io), [Twitter](https://twitter.com/leemunroe), [GitHub](https://github.com/leemunroe) | #ux-wg | [Google Group](https://groups.google.com/a/dcos.io/forum/#!forum/ux-wg) | TBD <br> [Join the Meeting](https://zoom.us/j/5214852707) <br> [Add to Google Calendar](https://calendar.google.com/calendar/hosted/mesosphere.io/event?action=TEMPLATE&hl=en&text=UX%20working%20group%20meeting&dates=20170303T093000%2F20170303T102000&location=SF%20%7C%20B%20%7C%20Manatee%20Nebula%20%287%20People%29%2C%20SF%20%7C%20B%20%7C%20Zoom%20%7C%20Manatee%20Nebula%20%287%20People%29&ctz=America%2FLos_Angeles&details=Join%20the%20meeting%3A%20https%3A%2F%2Fzoom.us%2Fj%2F5214852707) |
-
-### New Working Groups
-Email [Joerg Schad](mailto:joerg@mesosphere.com) to propose a new working group.
-
-## DC/OS Office Hours
-Every Thursday, join our community manager, [Judith Malnick](mailto:jmalnick@mesosphere.io), and other community members for office hours. Join the [Meetup](https://www.meetup.com/DC-OS-Online-Meetup/) to register for upcoming sessions, or check out the [DC/OS YouTube channel](https://www.youtube.com/playlist?list=PLVWqoBEzghqdpHYXcOESRPLJseA6cRT_q) to see previously recorded sessions.
+Welcome to the DC/OS community repo! Here you can learn how to get involved with the community. Most information is in the [Wiki](https://github.com/dcos/community/wiki), including a list of current working group meetings, FAQ's about DC/OS, and information about the Community team's work. Please feel free to explore and add any information you feel is missing.
```

Comment:
Bit of a nitpick, but it's just FAQ (Frequently Asked Questions), not FAQ's :)

Reply:
Lol great! Thanks!